apps/src/applab/designMode.js

```diff
@@ -564,7 +564,8 @@ designMode.readProperty = function(element, name) {
 designMode.onDuplicate = function(element, event) {
   let isScreen = $(element).hasClass('screen');
   if (isScreen) {
-    return duplicateScreen(element);
+    const newScreenId = duplicateScreen(element);
+    return elementUtils.getPrefixedElementById(newScreenId);
```

Comment:
`onDuplicate()` is supposed to return an element. We need TypeScript!

Reply:
Or [Flow](https://flow.org/)?  I think TS is a lot more popular, but a bit more challenging to introduce into our existing code.